### PR TITLE
CWPT-746 | improvement: Some improvements in the package

### PR DESCRIPTION
**Pull request by vansham69420**
- Implemented Prettier
- More linting rules
- Included the .env files in the dist folder.
- Removed src folder during build and packaging
- Error handling in the `generateName` function

### CODE REVIEW

**Review by prashantSyngenta — APPROVED** [review]

**Review by himanshu-2764 — APPROVED** [review]

File: src/modules/env/.env.production [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# -------------------------------------------------------
+# CROPWISE PLANTING DEV CONFIGURATION FOR PROD ENVIRONMENT
```
Comment (RambhauWaykar) [inline src/modules/env/.env.production:2]: why 2 env files for production?

File: src/modules/env/.env.production [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+		"Identity": "IDENTITYNOTSET",
+		"Timestamp": "TIMESTAMPNOTSET",
+		"Stage": "STAGENOTSET"
+	}'
+
+
+remoteConfig='{
+		"baseUrl":"https://cropwise-api-node-css-config.azurewebsites.net",
+		"environment": "production"
+    "password":"0HOetynAQ7zG"
```
Comment (RambhauWaykar) [inline src/modules/env/.env.production:26]: why this password here? can we store it in secret manager?

**Review by RambhauWaykar — CHANGES_REQUESTED** [review]

File: src/modules/env/.env.production (hunk 1 above)
Comment (vansham69420) [reply src/modules/env/.env.production:2]: azure uses the env name `production` while in AWS we are going ahead with `prod`. Hence to support the dual deployment to Azure and AWS. Also this is a temporary measure. Once migration to AWS is done, `production` and `staging` files are null and void.

File: src/modules/env/.env.production (hunk 2 above)
Comment (vansham69420) [reply src/modules/env/.env.production:26]: Sure

**Review by RambhauWaykar — APPROVED** [review]

**Pull request merged**